server/server.js

```diff
@@ -97,11 +101,14 @@ app.get('/trees', function (req, res) {
       query = {
         text: sql
       }
-      console.log(query);
+      console.log('query:', query);
 
     } else {
 
       console.log('Calculating clusters directly');
+      if (userid) { // make sure we get all the user’s trees
+        boundingBoxQuery = '';
```

Comment:
Does this mean you'll get trees outside the current view?

Reply:
Yes. If this isn't the desired behavior, I can change it back and just display a message saying “The user with id 123 has no trees in the current view.”